front/src/applications/editor/tools/pointEdition/types.ts

```diff
@@ -11,6 +11,8 @@ export type SignalingSystem = {
   | {
       signaling_system?: 'BAL';
       settings?: { Nf: 'true' | 'false' };
+      default_parameters?: { jaune_cli: 'true' | 'false' };
+      conditional_parameters: { on_route: string; parameters: { jaune_cli: 'true' | 'false' } }[];
```

Comment:
The code in utils is quite complex because the type is not very rigorous. You can try to use this type instead:
```
  export type SignalingSystem =
  | ({
      next_signaling_systems: Array<string | undefined>;
    } & (
      | {
          signaling_system: 'BAL';
          settings: { Nf: 'true' | 'false' };
          default_parameters: { jaune_cli: 'true' | 'false' };
          conditional_parameters: {
            on_route: string;
            parameters: { jaune_cli: 'true' | 'false' };
          }[];
        }
      | {
          signaling_system: 'BAPR';
          settings: { Nf: 'true' | 'false'; distant: 'true' | 'false' };
        }
      | {
          signaling_system: 'TVM';
          settings: { is_430: 'true' | 'false' };
        }
    ))
  | Record<string, never>;
```
This type also have a Record<string, never> (which is == `{}`) to represent the new empty object.